test/gui/shared/steps/steps.py

```diff
@@ -71,6 +71,10 @@ def step(context, displayname, host):
             ), displayname + "\n" + host
         )
 
+def startClient():
+    startApplication("owncloud -s --logfile - --language en_US.utf8 --confdir " + confdir)
+    snooze(1)
```

Comment:
The client takes language arguments like `en_US` not `en_US.utf8` .
@fmoc please add the format to the command line help text, see https://github.com/owncloud/client/issues/8574 .

Reply:
@TheOneRing fixed in #8580